src/test/resources/topic-response-bad-category.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "A public topic",
+  "id": 12345,
+  "category_id": 13
+}
```

Comment:
Could this have been done in the test case with setGetResponseData() instead of creating another file?